translationRecorder/app/src/main/java/org/wycliffeassociates/translationrecorder/ProjectManager/tasks/resync/ProjectListResyncTask.java

```diff
@@ -130,20 +148,20 @@ public ProjectListResyncTask(int taskId, Context ctx, FragmentManager fm) {
 
     @Override
     public void run() {
-        ProjectDatabaseHelper db = new ProjectDatabaseHelper(mCtx);
         Map<Project, File> directoriesOnFs = getProjectDirectoriesOnFileSystem();
         //if the number of projects doesn't match up between the filesystem and the db, OR,
         //the projects themselves don't match an id in the db, then resync everything (only resyncing
         // projects missing won't remove dangling take references in the db)
         //NOTE: removing a project only removes dangling takes, not the project itself from the db
-        if (directoriesOnFs.size() != db.getNumProjects() || db.projectsNeedingResync(directoriesOnFs.keySet()).size() > 0) {
-            fullResync(db, directoriesOnFs);
+        if (directoriesOnFs.size() != db.getNumProjects() || db.projectsNeedingResync(
+                directoriesOnFs.keySet()
+        ).size() > 0) {
```

Comment:
I know you're meaning to break apart the method call in projects needing resync, but you want to make sure that it lines up vertically when you do that, especially with its beginning and the close parenthesis

```
        if (
            directoriesOnFs.size() != db.getNumProjects() 
                || db.projectsNeedingResync(directoriesOnFs.keySet()).size() > 0
         ) {
```
is probably what you would do here? but realistically, you'd rather assign variables

```
boolean projectCountDiffers = directoriesOnFs.size != db.getNumProjects;
boolean projectsNeedResync = db.projectsNeedingResync(directoriesOnFs.keySet()).size > 0
if(projectCountDiffers || projectsNeedResync) {
```

added benefit here is the if statement is more readable as to its intent